### ISSUE TITLE
Problem with reverse DNS

### ISSUE THREAD

**Issue by sampaio23** [LukeSmithxyz/emailwiz#37]
Hi!
Looks like I'm having a problem with rDNS. My VPS is at Vultr and I use Epik Domain Registrar.
I can receive e-mails from other domains and I can send e-mail to myself, but when I try to send to gmail/msn (the ones I tested), it does not get there.

When trying _smtp:mydomain.xyz_ in [mxtoolbox](https://mxtoolbox.com/SuperTool.aspx), I get everything ok but one:

SMTP Reverse DNS Mismatch | Reverse DNS does not contain the hostname

My reverse DNS at Vultr is set to my-ip-address.mydomain.xyz.
Does anyone know what it can be? I don't get why I have to CNAME mail.mydomain.xyz to mydomain.xyz too.
Thanks! Best regards!

**Comment by sampaio23** [comment]
I see that there are some issues about it.
I'm gonna take a look and, if can't solve, I reopen!
Thanks!

**Comment by sampaio23** [comment]
I don't think this warning at mxtoolbox is a problem. Actually, I think the problem was with the VPS provider (I'm using Vultr). 

By default, they block port 25 (used by the SMTP), so looking at log files at `/var/log/mail.log`, I saw the connection was timing out, but when I tried sending a mail to myself, it worked because it didn't need to connect through this port.

Now, I asked them through Client Support to open it, and I'm waiting to see if it will work!
